common/biovotion/biovotion_vsm_energy.avsc

```diff
@@ -0,0 +1,12 @@
+{
+  "namespace": "org.radarcns.biovotion",
+  "type": "record",
+  "name": "BiovotionVSMEnergy",
+  "doc": "Energy expenditure data calculated by biovotion device.",
+  "fields": [
+    {"name": "time", "type": "double", "doc": "device timestamp in UTC (s)"},
+    {"name": "timeReceived", "type": "double", "doc": "device receiver timestamp in UTC (s)"},
+    {"name": "EnergyExpenditureValue", "type": "float", "doc": "Energy expenditure value (cal/s)"},
```

Comment:
would it be ok to use lower camel case for field names? And perhaps omit Value from the end?